src/data/cosmos_chains.json

```
@@ -270,7 +270,7 @@
       "id": "teritori",
       "name": "Teritori",
       "network_id": "teritori",
-      "image": "/images/chains/teritori.svg"
+      "image": "/images/chains/teritori.png"
```

```suggestion
      "image": "/images/chains/teritori.svg"
```
thanks. Should add an svg too @kalidax

---

Yes it should be svg not png

---

I only had PNG at the time :)